packages/graphql-auth-transformer/src/__tests__/__snapshots__/OperationsArgument.test.ts.snap

```diff
@@ -420,14 +432,20 @@ exports[`Test "create", "update", "delete" auth operations 5`] = `
   #set( $condition.expressionValues = $expressionValues )
 #end
 #if( $context.args.condition )
-  #set( $conditionFilterExpressions = $util.parseJson($util.transform.toDynamoDBFilterExpression($context.args.condition)) )
+  #set( $conditionFilterExpressions = $util.parseJson($util.transform.toDynamoDBConditionExpression($context.args.condition)) )
   $util.qr($condition.put(\\"expression\\", \\"($condition.expression) AND $conditionFilterExpressions.expression\\"))
   $util.qr($condition.expressionNames.putAll($conditionFilterExpressions.expressionNames))
   #set( $conditionExpressionValues = $util.defaultIfNull($condition.expressionValues, {}) )
   $util.qr($conditionExpressionValues.putAll($conditionFilterExpressions.expressionValues))
   #set( $condition.expressionValues = $conditionExpressionValues )
   $util.qr($condition.expressionValues.putAll($conditionFilterExpressions.expressionValues))
 #end
+#if( $condition.expressionValues && $condition.expressionValues.size() == 0 )
+  #set( $condition = {
```

Comment:
For the object copy we can do that, the if still needed, because we only have to remove ```expressionValues``` when it is empty, otherwise DynamoDB will error out, updated the code.